prelude/__internal__/IO.mad

```diff
@@ -2,14 +2,141 @@ import type { Wish } from "Wish"
 import { Wish } from "Wish"
 import IOError from "IOError"
 
+import {} from "Show"
+import IO from "IO"
+import {mapWithIndex} from "List"
```

Comment:
Nit but we need spaces between braces and names for consistency.

Reply:
I'd argue that until we have a linter which is enforced, this is a bike-shed problem.

But addressed nonetheless.